setup.sh

```diff
@@ -3,38 +3,57 @@
 LIGHT_GREEN='\033[1;32m'
 GREEN='\033[0;32m'
 CLEAR='\033[0m'
-echo -e "${CLEAR}${LIGHT_GREEN}Install prerequisites${CLEAR}"
 
-# Fedora
-if [ -n "$(command -v dnf)" ]; then
-    sudo dnf -y install curl git mercurial make binutils bison gcc glibc-devel
-fi
+function install_packages() {
+  echo -e "${CLEAR}${LIGHT_GREEN}Install prerequisites${CLEAR}"
 
-# Ubuntu
-if [ -n "$(command -v apt-get)" ]; then
-    sudo apt-get -y install curl git mercurial make binutils bison gcc build-essential
-fi
+  # Fedora
+  if [ -n "$(command -v dnf)" ]; then
+    sudo dnf -y install curl git mercurial make binutils bison gcc glibc-devel which
```

Comment:
Do we really need all of those packages?

Reply:
I don't know - It has been already there :-p 
I'll try to minimize the stack ;-)

Reply:
minimized the stack of packages as part of 3817dbe